bin/make-pot.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+wp i18n make-pot . --include"included-*.php" --skip-js
```

Comment:
I saw this article and thought this can be executed as a bash script: https://n8finch.com/wrote-first-bash-script-implement-wp-cli-managed-sites

Reply:
open to smth better

Reply:
related: https://developer.wordpress.org/block-editor/developers/internationalization/#create-translation-file

Reply:
`npm` script should be enough if WP-CLI is installed.